sdk/postgresql/arm-postgresql-flexible/review/arm-postgresql-flexible.api.md

```diff
@@ -2016,6 +2016,12 @@ export interface ServerThreatProtectionSettings {
     listByServer(resourceGroupName: string, serverName: string, options?: ServerThreatProtectionSettingsListByServerOptionalParams): PagedAsyncIterableIterator<ServerThreatProtectionSettingsModel>;
 }
 
+// @public
+export interface ServerThreatProtectionSettingsCreateOrUpdateHeaders {
+    // (undocumented)
+    location?: string;
+}
+
```

Comment:
Only this change? no api version change?

Reply:
yes

Reply:
could you check where to use this interface? could you help pin me this operation definition?

Reply:
related code in sdk: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/postgresql/arm-postgresql-flexible/src/operations/serverThreatProtectionSettings.ts#L339-L359

related code in swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-06-01-preview/ThreatProtection.json#L126-L197